api/contrib/envoy/extensions/matching/input_matchers/hyperscan/v3alpha/hyperscan.proto

```diff
@@ -14,14 +14,9 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: Hyperscan matcher]
 // [#extension: envoy.matching.input_matchers.hyperscan]
 
-// `Hyperscan <https://github.com/intel/hyperscan>`_ regex matcher. The engine of the matcher uses
-// hybrid automata techniques to allow simultaneous matching of large numbers of regular
-// expressions across streams of data.
-//
-// The matcher follows PCRE pattern syntax, and the regex string must adhere to the documented
```

Comment:
The paragraph has been moved to  `api/contrib/envoy/extensions/regex_engines/hyperscan/v3alpha/hyperscan.proto` since it is the introduction of the engine itself.